src/main/java/com/dwolla/rsocket/LoadBalancerBuilder.java

```diff
@@ -15,8 +15,9 @@
 
 import static org.asynchttpclient.Dsl.asyncHttpClient;
 import static org.asynchttpclient.Dsl.config;
+import java.io.IOException;
 
-public class LoadBalancerBuilder {
+public class LoadBalancerBuilder implements AutoCloseable {
```

Comment:
I guess I wouldn't expect the builder to close its `HttpClient`, since the `LoadbalanceRSocketClient` built by the builder (and using the `HttpClient`) is likely to outlive the builder? As long as the `LoadbalanceRSocketClient` is `AutoCloseable` I'd expect it will do what we want.